docs/intro/serverless.md

```diff
@@ -10,46 +10,78 @@ Schema and resolvers are created in-code and are passed to an Apollo GraphQL or
 
 ## Adding runtime layer to your application
 
-To create GraphQL Layer at runtime developers need to initialize `GraphbackRuntime` instance as follows:
+The `buildGraphbackAPI` method creates a schema, resolvers and CRUD services at runtime using your data model and some minimal configuration. The example below shows how you can use Graphback with Apollo and a Postgres database.
```

Comment:
Added documentation on how to use the new API. In general the website docs are outdated as they still refer to multiple ways of getting started.